### PR TITLE
Updated http script - added downloading support

### PR DESCRIPTION
**Pull request by Itay4**
Fixes https://github.com/demisto/server/issues/9685

### CODE REVIEW

File: Scripts/script-http.yml [hunk 1]
```diff
@@ -38,6 +39,16 @@ script: |+
   var insecure = (args.insecure === 'true');
   var proxy = (args.proxy === 'true');
   var res = http(args.url, requestObj, insecure, proxy);
+  var fileName = (res.Headers['Content-disposition']) ? res.Headers['Content-disposition'][0] : args.filename;
```
Comment (orius123) [inline Scripts/script-http.yml]: Shouldn't user argument override the headers?

File: Scripts/script-http.yml [hunk 2]
```diff
@@ -38,6 +39,22 @@ script: |+
   var insecure = (args.insecure === 'true');
   var proxy = (args.proxy === 'true');
   var res = http(args.url, requestObj, insecure, proxy);
+  if (args.filename) {
```
Comment (orius123) [inline Scripts/script-http.yml]: You should define `filename` outside the `if`

File: Scripts/script-http.yml [hunk 3]
```diff
@@ -38,6 +39,23 @@ script: |+
...
+      fileName = args.filename;
+  } else if (res.Headers['Content-disposition']) {
+      fileName = res.Headers['Content-disposition'][0];
+  } else {
+      fileName = 'http-file';
+  }
+
+  if (args.saveAsFile === 'yes') {
+      return {
```
Comment (Spivaka) [inline Scripts/script-http.yml]: are we sure that this is a secured file (3) and not a standard one (9)?

**Comment by Spivaka** [comment]
please fix conflicts and merge

File: Scripts/script-http.yml [hunk 4]
```diff
@@ -2,7 +2,8 @@ commonfields:
   id: http
   version: -1
 name: http
-releaseNotes: "-"
+description: Sends http request. Returns the response as json.
```
Comment (orius123) [inline Scripts/script-http.yml]: There is no `description` key for scripts. you should use `comment`

**Comment by dorsha** [comment]
@Itay4 - what about this PR?

**Pull request merged**